ansible/ansible-deploy-staging.yaml

```diff
@@ -27,6 +27,10 @@
       local_action: stat path="templates/{{ indicator }}-params-prod.json.j2"
       register: template
 
+    - name: Check to see if we have a secrets template to send.
+      local_action: stat path="templates/{{ indicator }}-secrets-prod.py.j2"
+      register: template
```

Comment:
Is this a name collision with `template` from above or is ansible smarter than that?

Reply:
Oh, yeah. It definitely could be. 🛠️

Reply:
Good catch!